code/modules/cargo/packs.dm

```diff
@@ -1387,13 +1387,6 @@
 					/obj/item/clothing/head/helmet/bluetaghelm)
 	crate_name = "laser tag crate"
 
-/datum/supply_pack/misc/lasertag/pins
-	name = "Laser Tag Firing Pins Crate"
-	cost = 3000
-	contraband = TRUE
-	contains = list(/obj/item/storage/box/lasertagpins)
```

Comment:
You should probably remove the storage item too considering it exists solely for the supply pack

Reply:
I have plans for the box in the future

Reply:
the future is not today, remove it